Configure

```diff
@@ -4865,18 +4865,21 @@ esac
 case "$ccname" in
     gcc)
 	$echo 'extern int foo;' > try.c
-	set X `$cppstdin -v try.c 2>&1 | $awk '/^#include </,/^End of search /'|$cppfilter $grep '/include'`
+	set X `$cppstdin -v try.c 2>&1 | $awk '/^#include </,/^End of search /'|$cppfilter $grep '/include$'`
```

Comment:
As a short test on my local linux boxes shows, this would change behavior:
```
$ gcc  -E -v try.c 2>&1 | /usr/bin/awk '/^#include </,/^End of search /'
#include <...> search starts here:
 /usr/lib64/gcc/x86_64-suse-linux/7/include
 /usr/local/include
 /usr/lib64/gcc/x86_64-suse-linux/7/include-fixed
 /usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/include
 /usr/include
End of search list.
```
As now `/usr/lib64/gcc/x86_64-suse-linux/7/include-fixed` would be skipped.

Likewise on HP-UX
```
 /wrk/ia64-4.7.2/bin/../lib/gcc/ia64-hp-hpux11.31/4.7.2/include
 /wrk/ia64-4.7.2/bin/../lib/gcc/ia64-hp-hpux11.31/4.7.2/include-fixed
 /usr/local/ia64/include
 /usr/include
```

What is the reason for adding the `$` EOL anchor? Is the **reason** to skip `include-fixed`?